scripts/azure-pipelines/linux-check.sh

```diff
@@ -73,7 +73,7 @@ if [[ "$DO_DOCS_CHECK" == "yes" ]] ; then
 fi
 
 if [[ "$DO_STYLE_CHECK" == "yes" ]] ; then
-    or_die make VERBOSE=1 check
+    or_die make VERBOSE=1 clangformat_check
```

Comment:
This was technically only doing clang format before because it was the only check executable in the image but this makes me feel better.